### PR TITLE
Add future template for pull requests

### PR DESCRIPTION
**Pull request by richiware**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Description
+<!--
+    Describe changes in detail.
+    This includes depicting the context, use case or current behavior and describe the proposed changes.
+-->
+
+## Contributor Checklist
+- [ ] Commit messages follow the project guidelines. <!-- External contributors should sign the DCO. Fast DDS developers must also refer to the internal Redmine task. -->
+- [ ] Any new/modified type has been properly reflected on `test/feature/dynamic_types/dds_types_tests` tests in a separate Pull Request. <!-- It is mandatory to test new/modified tests with the Dynamic Type API -->
```
Comment (JLBuenoLopez) [inline .github/pull_request_template.md]: Maybe a comment indicating that if a new IDL is introduced in the PR, the `utils/scripts/update_generated_code_from_idl.sh` script should also be updated correspondingly.

**Review by JLBuenoLopez — CHANGES_REQUESTED** [review]
Just a nitpick. Regarding the Python script I trust @adriancampo criteria

File: .github/pull_request_template.md (hunk 1 above)
Comment (richiware) [reply .github/pull_request_template.md]: Done in abcc155

**Review by JLBuenoLopez — APPROVED** [review]
LGTM

**Pull request merged**